index.html

```diff
@@ -0,0 +1,160 @@
+                        </li>
+                    </ul>
+                    <div class="d-flex justify-content-center">
+                        <button type="button" class="btn btn-outline-primary nav-btn1"> Get In Touch</button>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="container-fluid main-bg">
+            <div class="container d-flex flex-column main-heading">
+                <h1 class="mt-3">"Hello! Sharing World"</h1>
+                <h1 class="display-1">Aanchal Global<br> Impex Expo <br>2021</h1>
+                <div class="w-75 p-2 bg-white border-2">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Dolores, autem fuga explicabo nobis
+                        dolorum ipsam vitae veniam blanditiis, eum quam et iusto porr</p>
+                </div>
+                <h2 class="pt-3 text-dark font-weight-bold"><strong>2021.10.15 ~ 16(FRI)</strong> </h2>
+                <h4 class="pt-2 text-secondary">@National Export Council of India, Export HUB and more </h4>
+            </div>
+
+        </section>
+        <section class="container-fluid products-box d-flex flex-column justify-content-center align-items-center">
+            <h2 class="mt-5 product-heading">Our Products</h2>
+            <div class="container p-5">
+                <div class="row row-cols-1 row-cols-md-3 d-none d-lg-flex d-md-flex" id="products-list">
+
+
+                </div>
+                <div class="row row-cols-1 row-cols-md-3 d-lg-none d-md-flex" id="products-list2">
+
+
+                </div>
+            </div>
+            <button type="button" class="btn btn-outline-primary mb-5 d-lg-none d-md-block nav-btn" id="load-product-button">
+                Load More Products
+            </button>
+        </section>
+
+        <section class="container-fluid bg-light d-flex flex-column justify-content-center align-items-center">
+            <div class="container py-5 d-flex flex-column align-items-center">
+                <h2 class="feature-post">Featured Posts</h2>
+                <div class="row gap-3 d-flex justify-content-center d-none d-lg-flex d-md-flex" id="featured-container">
+
+
+                </div>
+                <div class="row gap-3 d-flex justify-content-center d-lg-none d-md-flex" id="featured-container2">
+
+                    
+                </div>
+            </div>
+            <button type="button" class="btn btn-outline-primary mb-5 d-lg-none d-md-block nav-btn" id="load-postt-button">
+                Load More Posts
+            </button>
+
+        </section>
+
+        <section class="container-fluid products-box d-flex flex-column justify-content-center">
+            <h2 class="mt-5 product-heading">Partner</h2>
+            <div class="container p-5">
+                <div class="partner">
```

Comment:
- [ ] The partner logos here look awesome on desktop. However, I would like you to consider changing it to the design as this screenshot from the design on mobile view:
![image](https://user-images.githubusercontent.com/68167810/132732586-a7f79208-7be8-40e0-b23d-551f8f78c376.png)